aea/mail/base.proto

```diff
@@ -1,6 +1,24 @@
 syntax = "proto3";
 
-package fetch.aea;
+package fetchai.aea;
```

Comment:
should be `package aea;` imo or maybe `aea.base` or something like that

Reply:
@marcofavorito Why not `aea.fetchai` to be coherent with the rest of protos?

Reply:
because `fetchai` refers to the author of the package/protocol. The base mail class has no author in that sense...